### PR TITLE
Feature/bump verdaccio default

### PR DESCRIPTION
**Pull request by jzolnowski**
- Change the default verdaccio version installed by the chart to 5.14.0
- Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,13 +82,13 @@ The following table lists the configurable parameters of the Verdaccio chart,
 and their default values.
 
 | Parameter                            | Description                                                                                                       | Default                        | Reference                                                                                                                 |
-| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------- | ------------------------------ | ------------------------------------------------------------------------------------------------------------------------- |
+| ------------------------------------ | ----------------------------------------------------------------------------------------------------------------- |--------------------------------| ------------------------------------------------------------------------------------------------------------------------- |
```
Comment (gkalpak) [inline README.md:85]: What is the purpose of this change?

**Review by gkalpak — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jzolnowski) [reply README.md:85]: Updated table with version of Verdaccio supported 😄

Comment (gkalpak) [reply README.md:85]: No, I am talking about this line in particular. You basically changed `| --...-- |` to `|---...---|` in one of the columns 🤔

Comment (jzolnowski) [reply README.md:85]: Oh, I'll fix that right away

**Review by gkalpak — APPROVED** [review]
LGTM :shipit:

**Comment by gkalpak** [comment]
@jzolnowski, please make sure all commits are signed.

**Review by Splaktar — APPROVED** [review]
